docs/labwc-config.5.scd

```diff
@@ -405,9 +411,9 @@ defined as shown below.
 </windowRules>
 ```
 
-*Actions*
+*Criteria*
 
-*<windowRules><windowRule identifier="" title="">*
+*<windowRules><windowRule identifier="" title="" singleton="">*
```

Comment:
`singleton` sounds very devel-centric. I am not sure what name to use instead though to make it understandable for non-developers. The feature itself sounds pretty useful.

Reply:
True.

I couldn't think of anything better. We could just call it `unique` or `singleInstance` or something.

Reply:
Just thinking out loud. `firstOnly` or `matchOnce`?

Reply:
Genius. `matchOnce` is very good.

Reply:
Done